core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3487,6 +3495,8 @@ private static String javadocLink(String fullyQualifiedClassname) {
         "alluxio.underfs.object.store.service.threads";
     public static final String UNDERFS_OBJECT_STORE_MOUNT_SHARED_PUBLICLY =
         "alluxio.underfs.object.store.mount.shared.publicly";
+    public static final String UNDERFS_OBJECT_STORE_MULTI_RANGE_CHUNK_SIZE_BYTES =
+        "alluxio.underfs.object.store.multi.range.chunk.size.bytes";
```

Comment:
nit: we used to have "bytes" in the end to indicate the unit. but a few releases ago, we are able to recognize conf values with different unites, like "256KB", "1MB", so I would not suffix "bytes" to the property name

Reply:
I think that's a good idea. I've seen similar things with a bunch of keys ending in `_MS` when clearly they don't need to be specified in milliseconds

Reply:
when you say user, do you mean developer or the user of Alluxio? if it is user, because we support "128KB", "1MB", it should be sufficient. I do think for developers, they may want to know the unit of the value they get. So I think in the future, we should add type to properties. So when they get `Property.UNDERFS_OBJECT_STORE_MULTI_RANGE_CHUNK_SIZE`, they know the unit of value they retrieved

Reply:
As a "user" I mean users of Alluxio. Someone who wants to configure Alluxio and wants to add this to their site properties will see the property `alluxio.underfs.object.store.multi.range.chunk.size`. But when reading this, to me, the word `size` doesn't indicate that we're looking to get a value expressed in bytes (or KB, MB< etc..).

Right in our code now we simply use `.getBytes()` and always return the configured value in pure bytes as a primitive long. I think that behavior is fine